packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -117,7 +116,7 @@ export class BlockList extends Component {
 					keyboardShouldPersistTaps="always"
 					scrollViewStyle={ { flex: isRootList ? 1 : 0 } }
 					data={ blockClientIds }
-					extraData={ [ isFullyBordered ] }
+					extraData={ [ true ] }
```

Comment:
@etoledom I have refactor the bordering logic according to convo with @iamthomasbishop and make some clean-up in the code.

However I'm wondering how to approach on this line ( I see that commit description i to prevent crash after merging gutenberg master to mobile ).

`isFullyBordered` flag was true in two scenarios:
1. For Blocks that holds InnerBlocks inside (it was hardcoded to true in `InnerBlock` component)
2. When the screen size was Mobile (the login in Layout component hold this value in state and pass it down)

So the false goes only to the RootList in only one case - in vertical position of the screen.

I believe that the purpose of that prop is to make sure `FlatList` will be rerender after state change.

According to all above the question is do you think we can hardcode it to true like above or we need to add some logic ?

Reply:
Hey @jbinda , thank for asking!

A bit of context about that merge commit: It was a big PR that solved a breaking change on the `gutenberg` repo. It needed a fair bit of refactoring.

This snippet you mention is not related to the crash itself, but it just came along as part of the refactoring, so changing it shouldn't have risks of a crash regression.

> So the false goes only to the RootList in only one case - in vertical position of the screen.

This is the case not only for phones on portrait mode, but on any case where the editor width was small enough. (Other example is multitasking on iPads).

As I see that we now, we want to always drawing the side lines, no matter the editor width. If that's the case then it sounds good to keep it always as true, or maybe we could remove this line completely since we won't have visual changes to re-render.

Reply:
> maybe we could remove this line completely since we won't have visual changes to re-render

That's what I was thinking. From what I understand, `extraData` is only used to decide whether the component needs to re-render. So it doesn't care about the actual value, only if it changes. If we always pass `true`, it seems equivalent to not passing it at all.

Reply:
@etoledom @koke 

Thanks for reply. I thought so this line won't be necessary but want to make sure according to @etoledom commit message about the reason behind mentioned crash.

So the conclusion is I will remove this line.

Apart of that do you agree with the rest of  the code cleanup that I did ?

Reply:
> extraData={ [ isFullyBordered ] }

That line was only necessary to get the view updated when border style changes, in cases like switching between Landscape & Portrait. If it is going to be a constant value we don't need it anymore.

Reply:
according to above conversation line was removed